ChangeLog.md

```diff
@@ -14,7 +14,7 @@ and this project adheres to
 * The `normalize` argument to `freud.density.RDF` is now `normalization_mode`.
 * The ``neighbors`` argument to ``env_neighbors`` for ``EnvironmentMotifMatch`` class.
 * The ``neighbors`` argument to ``cluster_neighbors`` for ``EnvironmentCluster`` class.
-* `freud.order.Nematic` uses orientation vectors instead of quaternions and a nematic director.
+* `freud.order.Nematic` uses orientation vectors instead of quaternions and a nematic director and raises warning when zero vector passed.
```

Comment:
```suggestion
* `freud.order.Nematic` uses orientation vectors instead of quaternions and a nematic director.
* `freud.order.Nematic` raises a warning when the zero is vector passed.
```